tests/blockchain/eth/contracts/integration/test_intercontract_integration.py

```diff
@@ -104,6 +106,30 @@ def policy_manager(testerchain, escrow):
     return contract, dispatcher
 
 
+@pytest.fixture()
+def overseer(testerchain, escrow):
+    escrow, _ = escrow
+    creator = testerchain.interface.w3.eth.accounts[0]
+
+    # secret_hash = testerchain.interface.w3.sha3(policy_manager_secret)
+
+    # Creator deploys the contract
+    #  TODO change to the normal contract
```

Comment:
What is required to change to the normal contract?

Reply:
After merging #509 contract will be ready to use in integration test. I plan to do this in #606